llama_stack/distribution/server/server.py

```diff
@@ -312,11 +312,13 @@ async def send_version_error(send):
         return await self.app(scope, receive, send)
 
 
-def main():
+def main(args: argparse.Namespace = None):
     """Start the LlamaStack server."""
     parser = argparse.ArgumentParser(description="Start the LlamaStack server.")
     parser.add_argument(
         "--yaml-config",
+        "--config",
+        dest="config",
```

Comment:
ah, much nicer :) 

wonder if we could instead add a `--config` argument separately, and add a deprecation warning for `--yaml-config`?

Reply:
Done!